### PR TITLE
Fix issue #34. Semantic kernel summaries plugin fails to preserve the original language of the text

### LINKED ISSUE #34: Semantic Kernel Summaries Plugin fails to preserve the original language of the text
### Current Behavior
The [summaries plugin](https://github.com/Encamina/enmarcha/blob/8393567191613b483fe07b5691787d767df825ba/src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/Plugins/TextPlugin/Summarize/skprompt.txt) is not preserving the original language of the text; it consistently returns the summary in English.

### Expected Behavior
The plugin should retain the original language of the text. For example, if the input text is in Spanish, the summary should be in Spanish.

### Tests
When requesting a summary for the following text, the plugin returns the summary in English.
#### Test data

- Model name: gpt-35-turbo
- Model version: 0613
- Input text: Augusta Ada King, condesa de Lovelace (Londres, 10 de diciembre de 1815-íd., 27 de noviembre de 1852), registrada al nacer como Augusta Ada Byron y conocida habitualmente como Ada Lovelace, fue una matemática y escritora británica, célebre sobre todo por su trabajo acerca de la computadora mecánica de uso general de Charles Babbage, la denominada máquina analítica. Fue la primera en reconocer que la máquina tenía aplicaciones más allá del cálculo puro y en haber publicado lo que se reconoce hoy como el primer algoritmo destinado a ser procesado por una máquina, por lo que se le considera como la primera programadora de ordenadores.
Lovelace fue la única hija legítima del poeta Lord Byron y Anna Isabella Noel Byron. Byron se separó de su esposa un mes después del nacimiento de Ada y dejó Inglaterra para siempre cuatro meses después. Conmemoró la despedida en un poema que comienza: «¿Es tu rostro como el de tu madre, mi bella hija? ¡ADA! Hija única de mi casa y mi corazón».​ Murió en la Guerra de independencia de Grecia cuando Ada tenía ocho años.
Dedujo y previó la capacidad de los ordenadores para ir más allá de los simples cálculos de números, mientras que otros, incluido el propio Babbage, se centraron únicamente en estas capacidades.
Su posición social y su educación la llevaron a conocer a científicos importantes como Andrew Crosse, sir David Brewster, Charles Wheatstone, Michael Faraday y al novelista Charles Dickens, relaciones que aprovechó para llegar más lejos en su educación. Entre estas relaciones se encuentra Mary Somerville, que fue su tutora durante un tiempo, además de amiga y estímulo intelectual.​ Ada Byron se refería a sí misma como una científica poetisa y como analista (y metafísica).​
> Text extracted from [Wikipedia](https://en.wikipedia.org/wiki/Ada_Lovelace)

#### Result
Ada Lovelace, born Augusta Ada Byron, was a British mathematician and writer known for her work on Charles Babbage's mechanical general-purpose computer, the Analytical Engine. She was the first to recognize that the machine had applications beyond pure calculation and published what is now recognized as the first algorithm intended to be processed by a machine, making her the first computer programmer. Lovelace was the only legitimate daughter of poet Lord Byron and Anna Isabella Noel Byron. Byron separated from his wife a month after Ada's birth and left England forever four months later. He commemorated their farewell in a poem. Lovelace deduced and predicted the capabilities of computers to go beyond simple number calculations, while others, including Babbage himself, focused solely on these capabilities. Her social position and education allowed her to meet important scientists such as Andrew Crosse, Sir David Brewster, Charles Wheatstone, Michael Faraday, and novelist Charles Dickens, relationships she 
used to further her education. One of these relationships was with Mary Somerville, who was her tutor, friend, and intellectual stimulus. Ada Byron referred to herself as a scientist poet and as an analyst (and metaphysician).

#### Expected Result
The summary of the text should be in Spanish (preserving the original language).

### Possible Solution:
After various tests modifying the prompt ([skprompt.txt](https://github.com/Encamina/enmarcha/blob/8393567191613b483fe07b5691787d767df825ba/src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/Plugins/TextPlugin/Summarize/skprompt.txt)) yielded unsatisfactory results, a potential solution could be to add a new parameter, "locale," to [config.json](https://github.com/Encamina/enmarcha/blob/8393567191613b483fe07b5691787d767df825ba/src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/Plugins/TextPlugin/Summarize/config.json). This parameter would explicitly specify the language in which the summary should be generated (thanks for the tip @rliberoff 😄).

### PR DESCRIPTION
**Pull request by LuisM000**
This is a pick from https://github.com/Encamina/enmarcha/pull/35 fixed in https://github.com/Encamina/enmarcha/issues/34.

### CODE REVIEW

**Comment by LuisM000** [comment]
I didn't notice this commented-out section
https://github.com/Encamina/enmarcha/blob/48e6f7508760648fde5678da64a0e167dabb99ff/.github/workflows/main.yml#L92
and the NuGets and Release have been generated 😞

File: src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/PluginsInfo.cs [hunk 1]
```diff
@@ -71,6 +71,11 @@ public static class Parameters
                     /// The name of the «minWordsCount» parameter, which represents the maximum number of words that the summary should have.
                     /// </summary>
                     public static readonly string MaxWordsCount = nameof(MaxWordsCount).ToLowerInvariant();
+
+                    /// <summary>
+                    /// The name of the «locale» parameter, which represents the language in which the summary will be generated.
```
Comment (rliberoff) [inline src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/PluginsInfo.cs:76]: There are some weird `Â` characters in the code. Could you please check them?

**Review by rliberoff — CHANGES_REQUESTED** [review]

File: src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/PluginsInfo.cs (hunk 1 above)
Comment (rliberoff) [reply src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/PluginsInfo.cs:76]: They also appear in the comments of the `MaxWordsCount` property.

Comment (rliberoff) [reply src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/PluginsInfo.cs:76]: image

**Review by rliberoff — APPROVED** [review]

**Pull request merged**